### ISSUE TITLE
Multiple examples is not supported

### ISSUE THREAD

**Issue by occupant23** [allure-framework/allure-java#201]
#### I'm submitting a ... 
  - [x] bug report
  - [ ] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

#### What is the current behavior?
Currently the test execution is crushing due to a NoSuchElementException when a Scenario Outline contains more than one Example (test data table). Due to the fact that only the first example is checked for a data set matching the current line in the test. This will lead to an exception when the script reaches the second example data.

```
[ERROR] Feature: Searching for products  Time elapsed: 2.596 s  <<< ERROR!
java.util.NoSuchElementException: No value present
```
#### If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal setup 

Run a feature with the following structure. 
```
@Smoke
Feature: Searching for products

  Scenario Outline: Searching for an existing product
    Given I am on the homepage of the Posters shop
    
    Then I want see the logo, the carousel and some hot products
    And the footer should be visible
    
    When I search for "<searchTerm>"
    Then I want to be on a category page
    And the page should show for the searchterm "<searchTerm>" "<expectedCount>" products
    
    When I click on the product number "<productPosition>"
    Then I want to be on a product detail page and see the "<productName>" as headline

    @Category1
    Examples: 
      | searchTerm | expectedCount | productPosition | productName                 |
      | bear       |             3 |               1 | Grizzly Bear                |

     @Category2
     Examples: 
      | searchTerm | expectedCount | productPosition | productName                 |
      | bee        |             9 |               4 | Indian Summer: Orange Beech |
```

#### What is the expected behavior?
The test should be executed without a failure since this was added to the cucumber standard quite some time ago.

#### What is the motivation / use case for changing the behavior?
Support the Cucumber features.

#### Please tell us about your environment:

| Allure version | 2.0-BETA22 |
| --- | --- |
| Test framework | JUnit@4.12 |
| Allure integration | allure-cucumber2-jvm@2.0-BETA20 |
| Generate report using | allure-maven@2.50 |

####  Diff for the fix
Class: `io.qameta.allure.cucumber2jvm.AllureCucumber2Jvm`

```
41c42
---
> import java.util.NoSuchElementException;
42c44
---
> import java.util.Iterator;
220,229c222,238
<         final Examples examples = scenarioOutline.getExamples().get(0);
<         final TableRow row = examples.getTableBody().stream()
<                 .filter(example -> example.getLocation().getLine() == currentTestCase.getLine())
<                 .findFirst().get();
<         return IntStream.range(0, examples.getTableHeader().getCells().size()).mapToObj(index -> {
<             final String name = examples.getTableHeader().getCells().get(index).getValue();
<             final String value = row.getCells().get(index).getValue();
<             return new Parameter().withName(name).withValue(value);
<         }).collect(Collectors.toList());
<     }
---
>         for (Iterator<Examples> iterator = scenarioOutline.getExamples().iterator(); iterator.hasNext();) {
>             final Examples examples = iterator.next();
>             final Optional<TableRow> oRow = examples.getTableBody().stream()
>                     .filter(example -> example.getLocation().getLine() == currentTestCase.getLine())
>                     .findFirst();
>             if (!oRow.isPresent()) {
>                 continue;
>             }
>             final TableRow row = oRow.get();
>             return IntStream.range(0, examples.getTableHeader().getCells().size()).mapToObj(index -> {
>                 final String name = examples.getTableHeader().getCells().get(index).getValue();
>                 final String value = row.getCells().get(index).getValue();
>                 return new Parameter().withName(name).withValue(value);
>             }).collect(Collectors.toList());
>         }
>         throw new NoSuchElementException();
>     }
```

**Comment by occupant23** [comment]
Hi there, we will change to version 2.8.1 in our projects next update since we want to support a newer Cucumber version.
While updating and testing I found that this issue is also present within the AllureCucumber3Jvm and can be fixed in the same manner as for AllureCucumber2Jvm. 
We already patched the file some time ago for our project since we need to support this feature. Your welcome to take the fix from us if it fulfills you coding standards.
[AllureCucumber2Jvm.java](https://github.com/Xceptance/neodymium-library/blob/master/src/main/java/io/qameta/allure/cucumber2jvm/AllureCucumber2Jvm.java)
